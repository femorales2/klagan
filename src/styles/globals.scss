:root {
  --font-mono: ui-monospace, Menlo, Monaco, 'Cascadia Mono', 'Segoe UI Mono',
    'Roboto Mono', 'Oxygen Mono', 'Ubuntu Monospace', 'Source Code Pro',
    'Fira Mono', 'Droid Sans Mono', 'Courier New', monospace;

  --navbar-bg: #000;
  --secondary-color: #fff;
  --marvel-red: #ec1d24;
  --scroll-track-bg: #d9d9d9;
}

* {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
}

a {
  display: flex;
}

html,
body {
  max-width: 100vw;
  overflow-x: hidden;
}
