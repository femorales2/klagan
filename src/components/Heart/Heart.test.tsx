import { describe, expect, it } from '@jest/globals';
import { render } from '@testing-library/react';
import Heart from '@/components/Heart/Heart';

describe('<Heart/>', () => {
  it('should match snapshot', () => {
    const {container} = render(<Heart/>);
    
    expect(container).toMatchSnapshot();
  });
})