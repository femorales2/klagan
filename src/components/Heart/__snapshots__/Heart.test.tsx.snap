// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heart/> should match snapshot 1`] = `
<div>
  <svg
    fill="none"
    height="12"
    viewBox="0 0 13 12"
    width="13"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M6.57153 2.37318L3.57153 0.552368L0.571533 2.37318V6.27491L6.57153 11.3905L12.5715 6.27491V2.37318L9.57154 0.552368L6.57153 2.37318Z"
      fill="#EC1D24"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
